lib/bridge.js

```diff
@@ -0,0 +1,165 @@
+			method : "POST",
+			body   : params
+		})
+		.then(function (response) {
+			var bridge = params;
+			var location = response.headers.location;
+			bridge.id = location.substring(location.lastIndexOf("/") + 1);
+			return bridge;
+		})
+		.asCallback(callback);
+	};
+
+	/**
+	 * Gets information about a bridge.
+	 * @param  {String} bridgeId The ID of the bridge to get
+	 * @param  {Function} callback A callback with the call information
+	 * @return {Promise} A promise for the call information
+	 */
+	this.get = function (bridgeId, callback) {
+		return client.makeRequest({
+			path   : "bridges/" + bridgeId,
+			method : "GET"
+		})
+		.then(function (response) {
+			return response.body;
+		})
+		.asCallback(callback);
+	};
+
+	/**
+	 * Gets a list of bridges.
+	 * @param {Object} params Query parameters for listing bridges
+	 * @param {Number} [params.page=0] Used for pagination to indicate the page requested for querying
+	 * a list of bridges.
+	 * If no value is specified the default is 0.
+	 * @param {Number} [params.size=25] Used for pagination to indicate the size of each page requested
+	 * for querying a list of bridges. If no value is specified the default value is 25 (maximum value 1000).
+	 * @param {Function} callback A callback with the list of bridges
+	 * @return {Promise} A promise for the list of bridges
+	 */
+	this.list = function (params, callback) {
+		return client.makeRequest({
+			path   : "bridges",
+			method : "GET",
+			qs     : params
+		})
+		.then(function (response) {
+			return response.body;
+		})
+		.asCallback(callback);
+	};
+
+	/**
+	 * Update the bridge
+	 * @param  {String} bridgeId The ID of the bridge
+	 * @param {Object} params Changed parameters of the bridge
+	 * @param {Boolean} params.bridgeAudio Enable/Disable two way audio path (default = true).
+	 * @param {Array<String>} params.callIds The list of call ids in the bridge.
+	 * @param {Function} [callback] Callback with the newly created bridge
+	 * @return {BridgeResponse} A promise for the operation
+	 */
+	this.update = function (bridgeId, params, callback) {
+		return client.makeRequest({
+			path   : "bridges/" + bridgeId,
+			method : "POST",
+			body   : params
+		})
+		.then(function () {
+			return;
+		})
+		.asCallback(callback);
+	};
+
+	function audioApi (bridgeId, params, callback) {
+		return client.makeRequest({
+			path   : "bridges/" + bridgeId + "/audio",
+			method : "POST",
+			body   : params
+		})
+		.then(function (response) {
+			return response.body;
+		})
+		.asCallback(callback);
+	}
+
+	/**
+	 * Speak sentence to the bridge
+	 * @param {String} bridgeId The ID of the bridge
+	 * @param {String} sentence A sentence to speak to the bridge.
+	 * @param {Object} params Optional parameters for the operation.
+	 * @param {Function} [callback] Callback for the operation
+	 * @return {BridgeResponse} A promise for the operation
+	 */
+	this.speakSentence = function (bridgeId, sentence, params, callback) {
```

Comment:
need to handle the case where no parameters are used and a callback is passed.

```bw.Bridge.speakSentence('123', 'hi how are you', function(err, res) {});``` 

will throw error:

```Unhandled rejection 
    at handleResponse (/Users/dtolbert/code/node-bandwidth/lib/client.js:21:10)
    at tryCatcher (/Users/dtolbert/code/node-bandwidth/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/dtolbert/code/node-bandwidth/node_modules/bluebird/js/release/promise.js:503:31)
    at Promise._settlePromise (/Users/dtolbert/code/node-bandwidth/node_modules/bluebird/js/release/promise.js:560:18)
    at Promise._settlePromise0 (/Users/dtolbert/code/node-bandwidth/node_modules/bluebird/js/release/promise.js:605:10)
    at Promise._settlePromises (/Users/dtolbert/code/node-bandwidth/node_modules/bluebird/js/release/promise.js:684:18)
    at Promise._fulfill (/Users/dtolbert/code/node-bandwidth/node_modules/bluebird/js/release/promise.js:629:18)
    at Request._callback (/Users/dtolbert/code/node-bandwidth/node_modules/bluebird/js/release/nodeback.js:42:21)
    at Request.self.callback (/Users/dtolbert/code/node-bandwidth/node_modules/request/request.js:199:22)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request.<anonymous> (/Users/dtolbert/code/node-bandwidth/node_modules/request/request.js:1036:10)
    at emitOne (events.js:101:20)
    at Request.emit (events.js:188:7)
    at IncomingMessage.<anonymous> (/Users/dtolbert/code/node-bandwidth/node_modules/request/request.js:963:12)
    at emitNone (events.js:91:20)```

This is how I handled that scenario for [endpoints](https://github.com/bandwidthcom/node-bandwidth/blob/endPoints/lib/endpoint.js#L63) but if you know a better way I'm open to suggestions